nucypher/blockchain/eth/actors.py

```diff
@@ -502,10 +543,18 @@ def initialize_stake(self,
 
         amount = NU(int(amount), 'NuNit')
 
+        current_period = self.miner_agent.get_current_period()
+        stake = Stake(owner_address=self.checksum_public_address,
+                      index=len(self.stakes)+1,
+                      start_period=current_period,
```

Comment:
In some cases `index` will be `len(self.stakes)` but first of all old unused stake slots will be used for new one (gas usage optimisation). So `index` is unused here, can we set it to `None` or something like this?  
Also `start_period` is `current_period` + 1

Reply:
:heavy_check_mark: